web/customize_viewer.css

```diff
@@ -55,3 +184,28 @@
     margin-left: 72px;
   }
 }
+
+/**
+ * Implement our custom icons. Disable all of the default icons, since the
+ * buttons now have inline svg in viewer.html.
+ */
+.toolbarButton::before {
+  content: '' !important;
+  display: none !important;
+}
+
+i.svg-icon {
+  margin: 0;
+  overflow: auto;
+}
+
+i.svg-icon path,
+i.svg-icon rect,
+i.svg-icon circle,
+i.svg-icon polygon,
+i.svg-icon ellipse,
+i.svg-icon polyline,
+i.svg-icon line,
+i.svg-icon g {
+  fill: currentColor;
```

Comment:
Update:

We talked through this and decided to leave it as-is for now. There are some problems with our current svg files that we'll need to fix in a future pull request.